### PR TITLE
COMMERCE-9279 follow up 2

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
@dsitu could you please have a look? thanks a lot

File: modules/apps/commerce/commerce-qualifier-service/src/main/java/com/liferay/commerce/qualifier/internal/metadata/CommerceBillingAddressCommerceQualifierMetadata.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+	extends BaseCommerceQualifierMetadata<Address> {
+
+	@Override
+	public Class<?> getConfigurationBeanClass() {
+		return AddressCommerceQualifierConfiguration.class;
+	}
+
+	@Override
+	public Predicate getFilterPredicate() {
+		ListType accountEntryAddressTypeBilling =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-qualifier-service/src/main/java/com/liferay/commerce/qualifier/internal/metadata/CommerceBillingAddressCommerceQualifierMetadata.java:57]:
```suggestion
		ListType accountEntryAddressTypeBillingListType =
```

File: modules/apps/commerce/commerce-qualifier-service/src/main/java/com/liferay/commerce/qualifier/internal/metadata/CommerceBillingAddressCommerceQualifierMetadata.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+			_listTypeLocalService.getListType(
+				AccountListTypeConstants.ACCOUNT_ENTRY_ADDRESS_TYPE_BILLING,
+				AccountListTypeConstants.ACCOUNT_ENTRY_ADDRESS);
+
+		ListType accountEntryAddressTypeBillingAndShipping =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-qualifier-service/src/main/java/com/liferay/commerce/qualifier/internal/metadata/CommerceBillingAddressCommerceQualifierMetadata.java:62]:
```suggestion
		ListType accountEntryAddressTypeBillingAndShippingListType =
```

File: modules/apps/commerce/commerce-qualifier-service/src/main/java/com/liferay/commerce/qualifier/internal/metadata/CommerceBillingAddressCommerceQualifierMetadata.java [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+			_listTypeLocalService.getListType(
+				AccountListTypeConstants.
+					ACCOUNT_ENTRY_ADDRESS_TYPE_BILLING_AND_SHIPPING,
+				AccountListTypeConstants.ACCOUNT_ENTRY_ADDRESS);
+
+		return AddressTable.INSTANCE.typeId.in(
+			new Long[] {
+				accountEntryAddressTypeBilling.getListTypeId(),
+				accountEntryAddressTypeBillingAndShipping.getListTypeId()
```
Comment (dsitu) [inline modules/apps/commerce/commerce-qualifier-service/src/main/java/com/liferay/commerce/qualifier/internal/metadata/CommerceBillingAddressCommerceQualifierMetadata.java:71]:
```suggestion
				accountEntryAddressTypeBillingListType.getListTypeId(),
				accountEntryAddressTypeBillingAndShippingListType.
					getListTypeId()
```

File: modules/apps/commerce/commerce-qualifier-service/src/main/java/com/liferay/commerce/qualifier/internal/metadata/CommerceShippingAddressCommerceQualifierMetadata.java [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+	extends BaseCommerceQualifierMetadata<Address> {
+
+	@Override
+	public Class<?> getConfigurationBeanClass() {
+		return AddressCommerceQualifierConfiguration.class;
+	}
+
+	@Override
+	public Predicate getFilterPredicate() {
+		ListType accountEntryAddressTypeShipping =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-qualifier-service/src/main/java/com/liferay/commerce/qualifier/internal/metadata/CommerceShippingAddressCommerceQualifierMetadata.java:57]:
```suggestion
		ListType accountEntryAddressTypeShippingListType =
```

File: modules/apps/commerce/commerce-qualifier-service/src/main/java/com/liferay/commerce/qualifier/internal/metadata/CommerceShippingAddressCommerceQualifierMetadata.java [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+			_listTypeLocalService.getListType(
+				AccountListTypeConstants.ACCOUNT_ENTRY_ADDRESS_TYPE_SHIPPING,
+				AccountListTypeConstants.ACCOUNT_ENTRY_ADDRESS);
+
+		ListType accountEntryAddressTypeBillingAndShipping =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-qualifier-service/src/main/java/com/liferay/commerce/qualifier/internal/metadata/CommerceShippingAddressCommerceQualifierMetadata.java:62]:
```suggestion
		ListType accountEntryAddressTypeBillingAndShippingListType =
```

File: modules/apps/commerce/commerce-qualifier-service/src/main/java/com/liferay/commerce/qualifier/internal/metadata/CommerceShippingAddressCommerceQualifierMetadata.java [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+			_listTypeLocalService.getListType(
+				AccountListTypeConstants.
+					ACCOUNT_ENTRY_ADDRESS_TYPE_BILLING_AND_SHIPPING,
+				AccountListTypeConstants.ACCOUNT_ENTRY_ADDRESS);
+
+		return AddressTable.INSTANCE.typeId.in(
+			new Long[] {
+				accountEntryAddressTypeShipping.getListTypeId(),
+				accountEntryAddressTypeBillingAndShipping.getListTypeId()
```
Comment (dsitu) [inline modules/apps/commerce/commerce-qualifier-service/src/main/java/com/liferay/commerce/qualifier/internal/metadata/CommerceShippingAddressCommerceQualifierMetadata.java:71]:
```suggestion
				accountEntryAddressTypeShippingListType.getListTypeId(),
				accountEntryAddressTypeBillingAndShippingListType.
					getListTypeId()
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 7]
```diff
@@ -180,10 +180,12 @@ account-eligibility=Account Eligibility
 account-entry-email-domains-configuration-name=Email Domains
 account-entry-group-configuration-description=Define which types of accounts are allowed on this site.
 account-entry-group-configuration-name=Accounts
+account-entry-qualifier-configuration-name=Account Qualifiers
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:183]:
```suggestion
account-entry-commerce-qualifier-configuration-name=Account Qualifiers
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 8]
```diff
@@ -180,10 +180,12 @@ account-eligibility=Account Eligibility
...
 account-group=Account Group
 account-group-filter-enabled=Account Group Filter Enabled
 account-group-ids=Account Group IDs
 account-group-name=Account Group Name
+account-group-qualifier-configuration-name=Account Group Qualifiers
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:188]:
```suggestion
account-group-commerce-qualifier-configuration-name=Account Group Qualifiers
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 9]
```diff
@@ -1064,6 +1066,7 @@ address-2=Address 2
 address-3=Address 3
 address-book=Address Book
 address-matches-shipment=Address Matches Shipment
+address-qualifier-configuration-name=Address Qualifiers
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:1069]:
```suggestion
address-commerce-qualifier-configuration-name=Address Qualifiers
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 10]
```diff
@@ -3415,16 +3420,20 @@ commerce-cart-content-total-display-context-help=Provides helper methods to retr
 commerce-cart-mini-web-portlet-instance-configuration-name=Commerce Mini Cart
 commerce-cart-total-web-portlet-instance-configuration-name=Commerce Cart Content Total
 commerce-cart-web-portlet-instance-configuration-name=Commerce Cart Content
+commerce-channel-qualifier-configuration-name=Commerce Channel Qualifiers
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:3423]:
```suggestion
commerce-channel-commerce-qualifier-configuration-name=Commerce Channel Qualifiers
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 11]
```diff
@@ -3415,16 +3420,20 @@ commerce-cart-content-total-display-context-help=Provides helper methods to retr
...
 commerce-currency-configuration-name=Exchange Rate Auto-Update
+commerce-currency-qualifier-configuration-name=Commerce Currency Qualifiers
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:3425]:
```suggestion
commerce-currency-commerce-qualifier-configuration-name=Commerce Currency Qualifiers
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 12]
```diff
@@ -3415,16 +3420,20 @@ commerce-cart-content-total-display-context-help=Provides helper methods to retr
...
 commerce-dashboard-forecast-portlet-instance-configuration-name=Commerce Dashboard Forecast Portlet Instance
 commerce-default-images-configuration-description=The fileEntryId of the image you wish to show when a product does not have an image.
 commerce-default-images-configuration-name=Commerce Default Images
 commerce-discount-configuration-name=Discounts
+commerce-discount-qualifier-configuration-name=Commerce Discount Qualifiers
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:3430]:
```suggestion
commerce-discount-commerce-qualifier-configuration-name=Commerce Discount Qualifiers
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 13]
```diff
@@ -3415,16 +3420,20 @@ commerce-cart-content-total-display-context-help=Provides helper methods to retr
...
 commerce-features-already-enabled-on-this-site=Commerce features already enabled on this site.
 commerce-geocoder-bing-configuration-name=Bing Geocoder
 commerce-instances-configuration-name=SKUs
 commerce-inventory-group-configuration-name=Commerce Inventory Group Configuration
 commerce-inventory-system-configuration-name=Commerce Inventory System Configuration
+commerce-inventory-warehouse-qualifier-configuration-name=Commerce Inventory Warehouse Qualifiers
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:3436]:
```suggestion
commerce-inventory-warehouse-commerce-qualifier-configuration-name=Commerce Inventory Warehouse Qualifiers
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 14]
```diff
@@ -3436,13 +3445,16 @@ commerce-order-importer-type-configuration-name=Order Importer Types
 commerce-order-items=Commerce Order Items
 commerce-order-items-help=Represents the list of commerce order items that is displayed.
 commerce-order-rule-entry-configuration-name=Order Rules
+commerce-order-rule-qualifier-configuration-name=Commerce Order Rule Qualifiers
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:3448]:
```suggestion
commerce-order-rule-commerce-qualifier-configuration-name=Commerce Order Rule Qualifiers
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 15]
```diff
@@ -3436,13 +3445,16 @@ commerce-order-importer-type-configuration-name=Order Importer Types
...
 commerce-order-type-configuration-name=Order Types
+commerce-order-type-qualifier-configuration-name=Commerce Order Type Qualifiers
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:3450]:
```suggestion
commerce-order-type-commerce-qualifier-configuration-name=Commerce Order Type Qualifiers
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 16]
```diff
@@ -3436,13 +3445,16 @@ commerce-order-importer-type-configuration-name=Order Importer Types
...
 commerce-orders=Commerce Orders
 commerce-organization-open-order-web-portlet-instance-configuration-name=Commerce Organization Open Order
 commerce-organization-order-display-context=Commerce Organization Order Display Context
 commerce-organization-web-portlet-instance-configuration-name=Commerce Organization Web Portlet Instance Configuration Name
 commerce-payment-method-authorize-net-configuration-name=Authorize.Net Payment Engine
 commerce-payment-method-authorize-net-group-service-configuration-name=Authorize.Net Payment Engine Group Service
+commerce-payment-method-group-rel-qualifier-configuration-name=Commerce Payment Method Group Rel Qualifiers
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:3457]:
```suggestion
commerce-payment-method-group-rel-commerce-qualifier-configuration-name=Commerce Payment Method Group Rel Qualifiers
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 17]
```diff
@@ -3451,6 +3463,7 @@ commerce-payment-method-offline-configuration-name=Offline Payment Method Keys
 commerce-payment-method-paypal-configuration-name=PayPal Payment Engine Method
 commerce-payment-method-paypal-group-service-configuration-name=PayPal Payment Engine Method Group Service
 commerce-price-entries-configuration-name=Price Entries
+commerce-price-list-qualifier-configuration-name=Commerce Price List Qualifiers
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:3466]:
```suggestion
commerce-price-list-commerce-qualifier-configuration-name=Commerce Price List Qualifiers
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 18]
```diff
@@ -3470,8 +3483,10 @@ commerce-product-price-range-facets-portlet-instance-configuration-name=Price Ra
 commerce-product-publisher-web-portlet-instance-configuration-name=Product Publisher
 commerce-shipping-engine-fedex-configuration-name=FedEx Shipping Engine
 commerce-shipping-engine-fedex-group-service-configuration-name=Fedex Shipping Engine Group Service
+commerce-shipping-fixed-option-qualifier-configuration-name=Commerce Shipping Fixed Option Qualifiers
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:3486]:
```suggestion
commerce-shipping-fixed-option-commerce-qualifier-configuration-name=Commerce Shipping Fixed Option Qualifiers
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 19]
```diff
@@ -3470,8 +3483,10 @@ commerce-product-price-range-facets-portlet-instance-configuration-name=Price Ra
...
 commerce-shipping-fixed-option-rel-info=Variable rate calculation: [Fixed Price] + ([Total Order Weight] * [Price per Unit of Weight]) + ([Order Subtotal] * [Subtotal Percentage])
 commerce-shipping-group-service-configuration-name=Shipping Group Service
+commerce-shipping-method-qualifier-configuration-name=Commerce Shipping Method Qualifiers
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:3489]:
```suggestion
commerce-shipping-method-commerce-qualifier-configuration-name=Commerce Shipping Method Qualifiers
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 20]
```diff
@@ -3480,6 +3495,7 @@ commerce-site-type=Commerce Site Type
 commerce-subscription-configuration-name=Subscriptions
 commerce-tax-by-address-type-configuration-name=Commerce Tax By Address
 commerce-term-entry-configuration-name=Commerce Terms
+commerce-term-entry-qualifier-configuration-name=Commerce Term Qualifiers
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:3498]:
```suggestion
commerce-term-entry-commerce-qualifier-configuration-name=Commerce Term Qualifiers
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**